app/controllers/v2/content_items_controller.rb

```diff
@@ -12,6 +14,18 @@ def index
       ).call
     end
 
+    def linkables
+      render json: Queries::GetContentCollection.new(
+        content_format: request.query_parameters.fetch(:format),
```

Comment:
Yeah not sure about this one.  The original was `content_format` but that was primarily to work around Rails nonsense which is no longer an issue.  There's other terms being used for the same thing so our conclusion was to defer this decision until this ticket: https://trello.com/c/1M8Njd0F/613-collating-the-names-for-everything